docs/docs/50-running/20-labels.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Labels and Annotations
+---
+
+As mentioned in the [authoring section](authoring/labels), you can add labels and annotations to your Acorns that will then be propogated down to the core Kubernetes resources created by Acorn. Authors of Acorn images can add these directly to the Acornfile, but users can also add them at runtime.
+
+The flags for adding labels and annotations allow you to specify the resource type and name you are targeting. This is best explained through examples:
+
+```shell
+
+# Add a label to all resources created by the app
+acorn run --label key=value
+
+# Add an annotation to just the top-level acorn app's metadata. No child resources will inherit
+acorn run --annotation metadata:key=value
+
+# Add a label to all resources created as part of any containers in the acorn
+acorn run --label containers:key=value
+
+# Add an annotation to the resources created as part of a specific container named 'mycontainer' in the acorn
+acorn run --annotation containers:mycontainer:key=value
+```
+
+Valid scopes are:
+- _none_ 
```

Comment:
how about we use `global` word instead of none to reflect that labels and annotations can be applied on all resources of the app ?

Reply:
I like it. will do